study_definition.md

```diff
@@ -425,6 +425,9 @@ household_size=patients.household_as_of(
 ),
 ```
 
+## Missing values
+
+If a query returns no matching record for a patient &mdash; for example if there are no blood pressure values recorded in a given period, or if there is no death date because the patient hasn't died, or if there is no household size available &mdash; then a default value will be returned. For strings and dates, the default value is the empty string `""`. For booleans, integers, or floats, the default value is `0`. There is no universal `null` value outputted to `input.csv`. 
```

Comment:
...and this is because handling nulls in CSVs is prone to all sorts of horrible errors (e.g. in pandas it assumes NA is a null value by default, which wreaks havoc when doing analysis of sodium pathology results....

(not sure if this needs explaining in the docs or not, I'll leave that to you!

Reply:
Oh good. I wasn't sure the "no null values" thing was true. I've added a small explanation